great_expectations/checkpoint/util.py

```diff
@@ -443,6 +442,17 @@ def get_updated_action_list(
     return list(base_action_list_dict.values())
 
 
+def batch_request_in_validations(
+    validations: Optional[List[dict]] = None,
+) -> bool:
+    if validations is not None:
+        for idx, val in enumerate(validations):
+            if val.get("batch_request") is not None:
+                return True
+
+    return False
```

Comment:
Why a util file? Can we make this a private staticmethod on the checkpoint where it is used?

Reply:
@cdkini It adds logically to other utility methods in this `util.py` file, so I would prefer to keep them there for now (till a formal refactoring effort, which I would agree with, given our latest style).  Thanks.